kube_scheduler/assets/dashboards/overview.json

```diff
@@ -1,642 +1,988 @@
+										{
+											"data_source": "metrics",
+											"name": "query1",
+											"query": "avg:kube_scheduler.max_fds{$cluster,$scope}"
+										},
+										{
+											"data_source": "metrics",
+											"name": "query2",
+											"query": "avg:kube_scheduler.open_fds{$cluster,$scope}"
+										}
+									],
+									"style": {
+										"palette": "dog_classic",
+										"line_type": "solid",
+										"line_width": "normal"
+									},
+									"display_type": "area"
+								}
+							],
+							"custom_links": []
+						},
+						"layout": {
+							"x": 6,
+							"y": 0,
+							"width": 6,
+							"height": 2
+						}
+					},
+					{
+						"id": 1253715625043501,
+						"definition": {
+							"title": "Goroutines",
+							"title_size": "16",
+							"title_align": "left",
+							"show_legend": false,
+							"legend_size": "0",
+							"type": "timeseries",
+							"requests": [
+								{
+									"response_format": "timeseries",
+									"queries": [
+										{
+											"data_source": "metrics",
+											"name": "query1",
+											"query": "avg:kube_scheduler.goroutines{$cluster,$scope}"
+										}
+									],
+									"style": {
+										"palette": "dog_classic",
+										"line_type": "solid",
+										"line_width": "normal"
+									},
+									"display_type": "line"
+								}
+							],
+							"custom_links": []
+						},
+						"layout": {
+							"x": 0,
+							"y": 2,
+							"width": 6,
+							"height": 2
+						}
+					},
+					{
+						"id": 1938988799424487,
+						"definition": {
+							"title": "Average go garbage collection duration",
+							"title_size": "16",
+							"title_align": "left",
+							"show_legend": false,
+							"legend_size": "0",
+							"type": "timeseries",
+							"requests": [
+								{
+									"response_format": "timeseries",
+									"queries": [
+										{
+											"data_source": "metrics",
+											"name": "query1",
+											"query": "avg:kube_scheduler.gc_duration_seconds.sum{$cluster,$scope}"
+										},
+										{
+											"data_source": "metrics",
+											"name": "query2",
+											"query": "avg:kube_scheduler.gc_duration_seconds.count{$cluster,$scope}"
+										}
+									],
+									"formulas": [
+										{
+											"formula": "derivative(query1) / derivative(query2)"
+										}
+									],
+									"style": {
+										"palette": "dog_classic",
+										"line_type": "solid",
+										"line_width": "normal"
+									},
+									"display_type": "area"
+								}
+							],
+							"custom_links": []
+						},
+						"layout": {
+							"x": 6,
+							"y": 2,
+							"width": 6,
+							"height": 2
+						}
+					}
+				]
+			},
+			"layout": {
+				"x": 0,
+				"y": 35,
+				"width": 12,
+				"height": 5
+			}
+		}
+	],
+	"template_variables": [
+		{
+			"name": "scope",
+			"default": "*",
+			"prefix": "",
+			"available_values": []
+		},
+		{
+			"name": "cluster",
```

Comment:
maybe it is better to use the full name here `kube_cluster_name`

Reply:
I used cluster to follow what was already in the cluster overview dashboard: https://app.datadoghq.com/dash/integration/30589/kubernetes-cluster-overview-dashboard